source/reference/system-variables.txt

```diff
@@ -43,27 +41,38 @@ Overview
          ``mongosqld`` events and state changes.
        - ``1``: Log messages for database users and ``mongosqld`` administrators.
        - ``2``: For internal use only.
-     - ``2``
-     - ``0``
+     - ``2 (Atlas-hosted) and 0 (on premises)``
    * - ``max_nested_table_depth``
      - integer
      - Specifies the maximum number of unique nested field paths that
        ``mongosqld`` maps to a relational table for a collection.
      - ``50``
-     - ``50``
-   * - ``max_num_columns_per_table``
+   * - ``max_num_global_tables``
+     - integer
+     - The maximum number of tables allowed in a table with an automatically-
+       generated schema.
+     - ``4000``
+   * - ``max_num_tables_per_collection``
+     - integer
+     - The maximum number of tables allowed in a collection in a table with an
+       automatically-generated schema.
+     - ``200``
+   * - ``max_nested_table_depth``
+     - integer
+     - The default limit for table's nesting depth in a table with an
```

Comment:
```suggestion
     - The default limit for table's nesting depth, that is the depth of arrays that will be translated to tables, in a collection with an
```

Reply:
Updated to become: "The default limit for table's nesting depth of arrays
       that MongoDB translates to tables, in a collection with an 
       automatically-generated schema."